src/Hamiltonians/ExtendedHubbardMom1D.jl

```diff
@@ -0,0 +1,128 @@
+"""
+    ExtendedHubbardMom1D(address; u=1.0, t=1.0, v=1.0, dispersion=hubbard_dispersion, boundary_condition = 0.0)
```

Comment:
```suggestion
    ExtendedHubbardMom1D(
        address; 
        u=1.0, t=1.0, v=1.0, dispersion=hubbard_dispersion, boundary_condition = 0.0
    )
```
Please break the long lines over multiple lines. The recommendation is to not exceed 72 characters per line to help readability. You can set vs-code to display a ruler in order to help you see quickly where the 72 character margin lies. See https://github.com/JuliaDiff/BlueStyle?tab=readme-ov-file#vs-code-settings